Dockerfile

```diff
@@ -30,11 +29,12 @@ RUN git config --global user.name "SUPERB Admin"
 ENV upstream_model osanseviero/hubert_base
 ENV downstream_task asr
 ENV hub huggingface
+ENV hf_hub_org None
```

Comment:
I've adopted the same approach we took for the `override` argument, namely to use a `None` string and check if is set in the runner